core/domain/fs_domain.py

```diff
@@ -44,16 +53,31 @@ def size(self):
 
 
 class FileStreamWithMetadata(object):
-    """A class that wraps a file stream, but adds extra attributes to it."""
+    """A class that wraps a file stream, but adds extra attributes to it.
+
+    Attributes:
+        content: blob. The content of the file snapshots.
```

Comment:
Here and elsewhere:
- Is "blob" actually a Python type? What do you get when you log `type(content)`?
- Why do you say "file snapshots" (plural) -- is there more than one?